src/object.c

```diff
@@ -874,9 +874,7 @@ size_t objectComputeSize(robj *key, robj *o, size_t sample_size, int dbid) {
         }
     } else if (o->type == OBJ_HASH) {
         if (o->encoding == OBJ_ENCODING_ZIPLIST) {
-            asize = sizeof(*o)+(ziplistBlobLen(o->ptr));
-        } else if (o->encoding == OBJ_ENCODING_LISTPACK) {
-            asize = sizeof(*o)+(lpBytes(o->ptr));
+            asize = sizeof(*o)+zmalloc_size(o->ptr);
```

Comment:
don't you need to handle OBJ_ENCODING_LISTPACK here too?
seems like a merge conflict that wasn't resolved properly...

Reply:
Ohh, my mistake.